### PR TITLE
Local networks improvements and other fixes

### PR DESCRIPTION
**Pull request by muresan**
### Summary and Scope

- Fixes:
- Requires:
- Relates to:

#### Issue Type

- Bugfix Pull Request
- Docs Pull Request
- RFE Pull Request

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Comment by manderson-hpe** [comment]
This PR requires `libxslt-tools` which we install on the hypervisor with the "mercury software proof-of-concepts" packages. Just an FYI note since it doesn't come with the default OS.

**Review by rustydb — APPROVED** [review]

File: dev/inventory/defaults.yaml [hunk 1]
```diff
@@ -107,16 +111,17 @@ volumes_defaults:
 node_defaults:
   vcpu: 2
   memory: 4096
-  network_interfaces:
-    - mode: macvtap
-      target: enp42s0
-      mtu: 1500
-      dhcp4: true
-      dhcp6: false
+  network_interfaces: []
```
Comment (manderson-hpe) [inline dev/inventory/defaults.yaml]: why did we remove the macvtap by default?

File: dev/inventory/local/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+keep
```
Comment (manderson-hpe) [inline dev/inventory/local/.gitkeep:1]: Do we have documentation on how to use this to override configs?

**Review by manderson-hpe — APPROVED** [review]
I tested this in my dev environment and it worked well.

File: dev/inventory/local/.gitkeep (hunk 2 above)
Comment (muresan) [reply dev/inventory/local/.gitkeep:1]: Once the code is stable, I will focus on documenting everything

File: dev/inventory/defaults.yaml (hunk 1 above)
Comment (muresan) [reply dev/inventory/defaults.yaml]: I was testing the airgapped setup. Reverted

**Pull request merged**